plugin/inc/menumonthlyplan.class.php

```diff
@@ -0,0 +1,30 @@
+<?php
+class PluginIserviceMenuMonthlyPlan extends CommonGLPI
+{
+
+    /**
+     * Right name used to check rights to do actions on item
+     *
+     * @var string
+     */
+    public static $rightname = 'entity';
+
+    public static function getMenuName(): string
+    {
+        return __('Plan lunar', 'iservice');
+    }
+
+    public static function getMenuContent(): array
+    {
+        if (!Session::haveRight(self::$rightname, READ)) {
+            return [];
+        }
+
+        return [
+            'title' => self::getMenuName(),
+            'page'  => Plugin::getPhpDir('iservice', false) . "/front/monthlyplan.php",
```

Comment:
We have `$CFG_PLUGIN_ISERVICE['root_doc']`
```suggestion
            global $CFG_PLUGIN_ISERVICE;
            ...
            'page'  => "$CFG_PLUGIN_ISERVICE[root_doc]/front/monthlyplan.php",
```
Anyway, I think `Plugin::getPhpDir` returns the path to the plugin directory on the server, not the link to the plugin.